app/assets/javascripts/sitewide/utils.js

```diff
@@ -88,10 +88,11 @@ $.fn.checkboxTreeLogic = function(dependencies, checkAll) {
  * submit gets JSON response, displays errors if any or either refreshes the
  * page or redirects it (if 'url' parameter is specified in JSON response).
  * @param  {string} modalID Modal ID
+ * @param  {string} modelName Modal Name
  * @param  {object} $fn     Link objects for opening the modal (can have more
  *         links for same modal)
  */
-$.fn.initializeModal = function(modalID) {
+$.fn.initSubmitModal = function(modalID, modelName) {
```

Comment:
What is the purpose of re-using of this old code?

Reply:
Not to write the same code again and again and to use sitewide utils.

Reply:
This code reloads the whole page, and for projects view we only reload project cards and breadcrumbs